javascript-bank/account.js

```diff
@@ -0,0 +1,60 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+// assign a new Transaction to a variable named newTransaction
+// set the type as 'deposit'
+// set the amount as the parameter for this function
+// check to make sure the amount is a positive, whole number
+// add the object to the transactions array in whichever account is being deposited to
+// return a boolean to show if the deposite was successfully added to the transactions array
+
+Account.prototype.deposit = function (amount) {
+  var newTransaction = new Transaction();
+  newTransaction.type = 'deposit';
+  newTransaction.amount = amount;
```

Comment:
You should be passing these values as arguments, not assigning the properties yourself in this function.